sites/star.azure.zooniverse.org.conf

```diff
@@ -0,0 +1,24 @@
+server {
+    include /etc/nginx/ssl.default.conf;
+    server_name "~^(?P<subdomain>.*)\.azure\.zooniverse\.org$";
+
+    location / {
+        resolver 8.8.8.8;
+        proxy_set_header       Host zooniversestatic.z13.web.core.windows.net;
+        proxy_http_version 1.1;
+
+        # This should point to static.zooniverse.org after the full DNS switch is complete
+        rewrite (?i)\.(jp(e)?g|gif|png|ico|txt|woff|ttf|mp(3|4)|webm|og(a|g|m|v|x)|spx|opus|pdf|tar|gz|tgz|bz2|tbz2|zip|js|css)$ https://zooniversestatic.z13.web.core.windows.net/$subdomain$request_uri;
+
+        add_header 'X-Content-Type-Options' 'nosniff';
+        add_header 'Content-Security-Policy' "frame-ancestors 'self'";
+        add_header 'X-Content-Security-Policy' "frame-ancestors 'self'";
+        add_header 'X-XSS-Protection' '1; mode=block';
+        proxy_cache            STATIC;
+        proxy_cache_valid      200  1d;
+        proxy_cache_use_stale  error timeout invalid_header updating
+                    http_500 http_502 http_503 http_504;
+
+        proxy_pass             https://zooniversestatic.z13.web.core.windows.net/$subdomain;
```

Comment:
I think you'll still want the `$request_uri` variable here to ensure any paths / query params are appended to the proxy request. https://github.com/zooniverse/static/blob/f0c2c013dfc6343bbba583512a7e6190768b2f17/sites/star.preview.zooniverse.org.conf#L17
```suggestion
        proxy_pass             https://zooniversestatic.z13.web.core.windows.net/$subdomain$request_uri;
```

Reply:
I'm wondering how the subdomain variable will proxy to the correct container path here. E.g. if the incoming request host headers is `alice.azure.zooniverse.org` this will proxy pass to the following URL path `https://zooniversestatic.z13.web.core.windows.net/alice$request_uri;`, right? 

However the paths in the azure containers are `$web/alice.zooniverse.org/index.html` etc.

So i think we need to modify the proxy pass url slightly, perhaps (string / variable interpolation will need testing)
```
proxy_pass             https://zooniversestatic.z13.web.core.windows.net/$subdomain.zooniverse.org$request_uri;
```

Reply:
Makes sense!